### PR TITLE
Add support for Chrome and Firefox web browsers on Windows

### PR DESCRIPTION
**Pull request by bellenot**

### CODE REVIEW

File: gui/webdisplay/src/TWebWindowsManager.cxx [hunk 1]
```diff
@@ -431,6 +449,15 @@ bool ROOT::Experimental::TWebWindowsManager::Show(ROOT::Experimental::TWebWindow
 
       TestProg(prog, gEnv->GetValue("WebGui.Firefox", ""));
 
+#ifdef _MSC_VER
+      TString ProgramFiles = gSystem->Getenv("ProgramFiles");
+      ProgramFiles.ReplaceAll(" (x86)", "");
+      TString ProgramFilesx86 = gSystem->Getenv("ProgramFiles(x86)");
+      if (ProgramFilesx86.Length() && !gSystem->AccessPathName(Form("%s\\Mozilla Firefox\\firefox.exe", ProgramFilesx86.Data())))
+         prog = TString::Format("%s\\Mozilla Firefox\\firefox.exe", ProgramFilesx86.Data());
```
Comment (linev) [inline gui/webdisplay/src/TWebWindowsManager.cxx]: Please use here call like:
if (ProgramFilesx86.Length())
    TestProg(prog, TString::Format("%s\\Mozilla Firefox\\firefox.exe", ProgramFilesx86.Data()));

Inside TestProg I check if executable is existing.
And if program already found (or configured by user) no new tests are required

File: gui/webdisplay/src/TWebWindowsManager.cxx [hunk 2]
```diff
@@ -415,10 +426,17 @@ bool ROOT::Experimental::TWebWindowsManager::Show(ROOT::Experimental::TWebWindow
       TestProg(prog, "/usr/bin/chrome-browser");
 #endif
       is_chrome = prog.Length()>0;
+#ifdef _MSC_VER
+      if (win.IsBatchMode())
+         exec = gEnv->GetValue("WebGui.ChromeBatch", "fork: --headless --disable-gpu --remote-debugging-port=8090 --enable-logging $url");
```
Comment (linev) [inline gui/webdisplay/src/TWebWindowsManager.cxx]: >   --remote-debugging-port=8090 

This is not very elegant solution. One other should try same workaround, like in Linux 

    --remote-debugging-socket-fd=0

or at least number should be configurable via gEnv

Comment (bellenot) [reply gui/webdisplay/src/TWebWindowsManager.cxx]: OK, will test again with `--remote-debugging-socket-fd=0`

File: gui/webdisplay/src/TWebWindowsManager.cxx (hunk 1 above)
Comment (bellenot) [reply gui/webdisplay/src/TWebWindowsManager.cxx]: OK, I will do it that way

**Pull request merged**